packages/zk-certificates/lib/keyManagement.ts

```diff
@@ -134,6 +134,31 @@ export function createHolderCommitment(
   ).toString();
 }
 
+/**
+ * Decompress an EdDSA public key.
+ * @param pubKeyHex - Compressed public key as hex string.
+ * @returns The decompressed public key as [Ax, Ay] strings in decimal format.
+ */
+export async function decompressEddsaPubKey(
+  pubKeyHex: string,
+): Promise<[string, string]> {
+  const babyjub = await buildBabyjub();
+  if (pubKeyHex.length !== 64) {
+    throw new Error('Invalid public key length');
+  }
+  const hexArray = pubKeyHex.match(/.{2}/gu) ?? [];
+  // Convert hex array to uint8 byte array
+  const pubKeyArray = new Uint8Array(
```

Comment:
There is a built-in function for this: https://nodejs.org/api/buffer.html#static-method-bufferfromstring-encoding

Reply:
I agree. Copy pasted this part from a time before the snap execution environment supported the buffer built-in.